### ISSUE TITLE
help doc request from Mary

### ISSUE THREAD

**Issue by nataliehamilton** [nhamiltonrally/hello-world#22]
help doc request: How to unarchive a card

just added a way to unarchive from the throughput card list : https://waffle.io/waffleio/waffle.io/metrics/throughput 

you will see some have a blue unarchive under the closed time

only team members will see this option

and it only shows if the card has been archived

this list is part of metrics, so is a paid feature for those with private repos

**Comment by nataliehamilton** [comment]
https://help.waffle.io/faq/done-column-closing-issues/can-i-view-my-archived-issues

**Comment by nataliehamilton** [comment]
Natalie [14 days ago]
@mary, will you review? I added it to the bottom of https://help.waffle.io/faq/done-column-closing-issues/can-i-view-my-archived-issues. I think this will be moved out of FAQ during the re-org, but does it make sense for now? I also want to add a GIF for the whole process described in this article.
help.waffle.io
Can I view my archived issues?
View archived issues and unarchive


mary [13 days ago]
thanks! someone has already used it! just a few notes:
1. this says “issues” but it works for “pull requests” too.
2. this sentence “that have been archived in the last 30 days” should say `closed` instead of `archived` because if they archive a card that was closed over 30 days ago it won’t be in that list
3. this sentence “Unarchive is a paid feature for Waffle users with private repos.” makes me think it isn’t free for public repos but that might just be me.


Natalie [12 days ago]
Thanks @mary! I’ll get this updated.
